lifecycle-application/src/test/java/io/quarkus/ts/lifecycle/AlphabeticallyFirstTest.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.ts.lifecycle;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class AlphabeticallyFirstTest {
```

Comment:
Make an explicit comment about the importance of the class name in relation to the `HelloMainTest` class for the test of the mixing of the `@QuarkusTest` and `@QuarkusMainTest` to work.

Maybe include a link to the PR.

Granted, it is mentioned in the PR comment, but that may not be very visible during a refactoring or so.